backend/google_cds_store/google_cds_store.go

```diff
@@ -182,7 +182,7 @@ func (s *googleCDSStore) GetSettings(ctx context.Context) (models.Settings, erro
 	var result models.Settings
 	err := s.client.Get(ctx, key, &result)
 	if err == datastore.ErrNoSuchEntity {
-		result = models.Settings{}
+		result, err = s.CreateAndGetSettings(ctx, key)
```

Comment:
We can probably just have an `UpsertSettings` method which updates settings in the database, creating a new settings object if not already present. That might be useful for other things later (e.g. updating the general settings).

If we're going to modify the behaviour of the `GetSettings` like this (which I think is a good idea) then we should probably do it in a separate PR.

Reply:
Okay, I've reset the file now.